src/database/models/ubi/beneficiary.ts

```diff
@@ -1,7 +1,19 @@
 import { User } from '@interfaces/app/user';
 import { Sequelize, DataTypes, Model } from 'sequelize';
+
 import { CommunityAttributes } from './community';
 
+export type IBeneficiaryActivities = {
```

Comment:
Move this type to https://github.com/impactMarket/api/blob/master/src/types/endpoints.ts, maybe next to `IListBeneficiary`. Lets keep within the models' folder only what belongs to models.

Reply:
done!